tests/PHPStan/Analyser/data/strtr.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Strtr;
+
+use function PHPStan\Testing\assertType;
+
+/**
+ * @param non-empty-string $nonEmptyString
+ * @param non-falsy-string $nonFalseyString
+ */
+function doFoo(string $s, $nonEmptyString, $nonFalseyString) {
```

Comment:
results before this PR:
https://phpstan.org/r/8833f01d-4202-4163-bf72-982f9729c3c9